v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/options/KafkaReadOptions.java

```diff
@@ -49,23 +49,49 @@ public interface KafkaReadOptions extends PipelineOptions {
 
   void setKafkaReadTopics(String inputTopics);
 
-  @TemplateParameter.Enum(
+  @TemplateParameter.Boolean(
       order = 3,
       groupName = "Source",
+      name = "enableCommitOffSets",
```

Comment:
"offset" is a single work so the parameter name should be "enableCommitOffsets" (OffSets -> Offsets).

Reply:
Done